src/main/common/streambuf.c

```diff
@@ -114,6 +114,7 @@ uint32_t sbufReadU32(sbuf_t *src)
 void sbufReadData(sbuf_t *src, void *data, int len)
 {
     memcpy(data, src->ptr, len);
+    src->ptr += len;
```

Comment:
We also have `sbufAdvance()` as this would affect following code as well

https://github.com/betaflight/betaflight/blob/3e130de49ce5ceffc0e38e85d0fdb5d06c07a6bd/src/main/msp/msp.c#L3980-L3994

Reply:
This code assumes that sbufReadData advances internal pointer.  `sbufAdvance(src, length - MAX_BOARD_NAME_LENGTH); ` only skips to end of entry if it does not fit MAX_BOARD_NAME_LENGTH